k8s/plugin/src/resources/uo_client.rs

```diff
@@ -0,0 +1,156 @@
+use crate::constant::UPGRADE_OPERATOR_END_POINT;
+use anyhow::Error;
+use http::Method;
+use hyper::body::Buf;
+use serde::{Deserialize, Serialize};
+/// Client to interact with upgrade operator.
+#[derive(Debug)]
+pub(crate) struct UpgradeOperatorClient {
+    /// Address of UpgradeOperatorClient service.
+    pub uri: Uri,
+    /// UpgradeOperatorClient client.
+    pub inner_client: kube_proxy::UpgradeOperatorClient,
+    /// Endpoint of upgrade operator service.
+    pub service_endpoint: Uri,
```

Comment:
Ah sorry, if this is a Uri, maybe the cli argument should be `Uri` instead of `Url` ?